components/domains/steps/checkoutCard.tsx

```diff
@@ -412,10 +412,14 @@ const CheckoutCard: FunctionComponent<CheckoutCardProps> = ({
     // If the user has toggled autorenewal
     if (renewalBox) {
       if (needsAllowance) {
+        const amountToApprove = String(
```

Comment:
It approves the price of 1 year, shouldn't it be 10 years ?

Reply:
Are you sure ? For me it works and send 10 years